src/arrayMethodSort.js

```diff
@@ -5,7 +5,31 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    if (compareFunction !== undefined) {
```

Comment:
try to pass a default value for the `compareFunction`
it will make your code shorter and without code duplications